README.md

```diff
@@ -1 +1,74 @@
-# setup-mobile-skeleton
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Description the project.
+
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Author1**
+ GitHub: [@rehanullah748](https://github.com/@rehanullah748)
+- Twitter: [Rehanul47235282](https://twitter.com/Rehanul47235282)
+- LinkedIn: [rehan-ullah-2b644b239](https://linkedin.com/in/rehan-ullah-2b644b239)
+
+👤 **Author2**
+
+- GitHub: [@rehanullah748](https://github.com/@rehanullah748)
+- Twitter: [Rehanul47235282](https://twitter.com/Rehanul47235282)
+- LinkedIn: [rehan-ullah-2b644b239](https://linkedin.com/in/rehan-ullah-2b644b239)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] Good job for proffesional README.
- [ optional] README file should be descriptive and all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present in a your repo and please try to customize.